src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -344,6 +344,22 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 // this is symmetric with context.res which we also support
                 context["req"] = requestObject;
             }
+            if (input is HttpResponseMessage)
```

Comment:
>if (input is HttpResponseMessage) [](start = 12, length = 33)

In the new proposal, I think we can get rid of this.  (I'm planning to get rid of the TimerInfo case below too)